packages/mock/src/mockMatchers.ts

```diff
@@ -51,6 +51,15 @@ const extension = {
     const nthCall = m.calls[n];
     return isEqual(nthCall.slice(0, args.length), args);
   },
+
+  toHaveReturned: (m: unknown, ...args: any[]): boolean => {
```

Comment:
Hmm, this logic seems to be the copy of `toHaveBeenCalled`, so probably this should be like this:
```suggestion
  toHaveReturned: (m: unknown, value: any): boolean => {
```
And we should search for `value` in `m.returns`, like in `toHaveBeenCalledWith`